app/scripts/controllers/replicaSet.js

```diff
@@ -311,42 +383,41 @@ angular.module('openshiftConsole')
           }
         );
 
-        watches.push(DataService.watch($scope.resource, context, function(deployments, action, deployment) {
-          $scope.deployments = deployments.by("metadata.name");
+        watches.push(DataService.watch($scope.resource, context, function(replicaSets, action, replicaSet) {
+          $scope.replicaSets = replicaSets.by("metadata.name");
           $scope.emptyMessage = "No deployments to show";
-          $scope.deploymentsByDeploymentConfig = DeploymentsService.associateDeploymentsToDeploymentConfig($scope.deployments);
+          $scope.deploymentsByDeploymentConfig = DeploymentsService.associateDeploymentsToDeploymentConfig($scope.replicaSets);
```

Comment:
is this a wasted call if these are actually replica sets and not replication controllers?